components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -464,13 +465,23 @@ class TabsUseCases(
     class DuplicateTabUseCase(
         private val store: BrowserStore
     ) {
+        /**
+         * Creates a duplicate of the currently selected tab (including history) and
+         * selects it if [selectNewTab] is true.
+         */
+        fun invoke(selectNewTab: Boolean): String? {
+            return store.state.selectedTab?.let {
+                this.invoke(it, selectNewTab)
+            }
+        }
+
         /**
          * Creates a duplicate of [tab] (including history) and selects it if [selectNewTab] is true.
          */
         operator fun invoke(
             tab: TabSessionState,
             selectNewTab: Boolean = true
-        ) {
+        ): String {
```

Comment:
Please update the docs to describe the new return value.

Reply:
Done.